### PR TITLE
Feature/header component

### PR DESCRIPTION
**Pull request by adrianacostep33**

### CODE REVIEW

File: frontend/src/components/button/Button.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export const StyledButton = styled.button<ButtonProps>`
+  cursor: pointer;
+  padding: ${rem(15)};
+  background-color: ${(props) =>
+    props.type === "text" ? "transparent" : `${pallete.PLATINUM} `};
+  border-radius: ${rem(10)};
+  color: ${(props) =>
+    props.type === "text" ? `${pallete.PLATINUM}` : `${pallete.SPACE_CADET}`};
+  font-size: ${rem(16)};
+  &:hover {
```
Comment (roxanap33) [inline frontend/src/components/button/Button.styled.tsx:19]: add a new line for better visibility?

File: frontend/src/components/button/Button.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    background-color: ${pallete.ULTRA_VIOLET};
+  }
+  @media (max-width: 600px) {
```
Comment (roxanap33) [inline frontend/src/components/button/Button.styled.tsx:23]: here as well

File: frontend/src/components/header/Header.tsx [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+            <img src={logo} alt="logo" />
+          </LogoContainer>
+          <InputContainer>
+            <input type="text" />
+            <SearchIcon>
+              <SearchOutlinedIcon />
+            </SearchIcon>
+          </InputContainer>
+        </LeftSide>
+
```
Comment (roxanap33) [inline frontend/src/components/header/Header.tsx]: delete this empty line

File: frontend/src/contexts/authContext.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { createContext, useState, useContext } from "react";
+
+interface AuthContextType {
```
Comment (roxanap33) [inline frontend/src/contexts/authContext.tsx:3]: maybe move this to another file and import it

File: frontend/src/contexts/authContext.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  name: string;
+  picture: string;
+  isAuthenticated: boolean;
+}
+
+interface AuthProviderProps {
```
Comment (roxanap33) [inline frontend/src/contexts/authContext.tsx:9]: move this too

File: frontend/src/components/button/Button.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import { ButtonContainer, StyledButton } from "./Button.styled";
+
+interface ButtonProps {
+  type: "primary" | "text";
+  children: React.ReactNode;
+}
+
+const Button = ({ type, children }: ButtonProps) => {
+  return (
+    <ButtonContainer>
```
Comment (ioanagirbap33) [inline frontend/src/components/button/Button.tsx]: Why do we need the wrapper around the button?

File: frontend/src/contexts/authContext.tsx (hunk 4 above)
Comment (adrianacostep33) [reply frontend/src/contexts/authContext.tsx:3]: I use AuthContextType just in this file

**Review by roxanap33 — APPROVED** [review]

**Review by ioanagirbap33 — APPROVED** [review]

**Pull request merged**